weaviate/collections/classes/config_methods.py

```diff
@@ -242,10 +242,10 @@ def _properties_from_config(schema: Dict[str, Any]) -> List[_Property]:
             ),
             vectorizer_config=(
                 _PropertyVectorizerConfig(
-                    skip=prop["moduleConfig"][schema["vectorizer"]]["skip"],
-                    vectorize_property_name=prop["moduleConfig"][schema["vectorizer"]][
+                    skip=prop["moduleConfig"][schema["vectorizer"]].get("skip"),
+                    vectorize_property_name=prop["moduleConfig"][schema["vectorizer"]].get(
                         "vectorizePropertyName"
-                    ],
+                    ),
```

Comment:
Since `skip` and `vectorize_property_name` are `bool`, can the default return here be `False` rather than `None`?